src/main/webapp/i18n/en/fileUploadExercise.json

```diff
@@ -6,9 +6,9 @@
                 "createLabel": "Create a new File Upload Exercise",
                 "editLabel": "Edit a File Upload Exercise"
             },
-            "created": "A new File Upload Exercise is created with identifier {{ param }}",
-            "updated": "A File Upload Exercise is updated with identifier {{ param }}",
-            "deleted": "A File Upload Exercise is deleted with identifier {{ param }}",
+            "created": "Created new File Upload Exercise with identifier {{ param }}",
+            "updated": "Updated File Upload Exercise with identifier {{ param }}",
+            "deleted": "Deleted File Upload Exercise with identifier {{ param }}",
             "delete": {
                 "question": "Are you sure you want to delete File Upload Exercise {{ title }}?",
                 "typeNameToConfirm": "Please type in the name of the Exercise to confirm."
```

Comment:
most of the times, the value for key "typeNameToConfirm" does not use capitalized nouns, e.g. "the name of the course / lecture / team ...". Maybe we can change the occurrences of this standard sentence to always _not_ capitalize the noun.

Reply:
I agree with you, and there are lots of other places where I would have liked to remove capitalization as well. There were so many instances of this though, with a lot of them falling into sort of gray areas where you could argue either way about it being a proper or improper noun that I figured I would deal with capitalization another time. I'll make a commit for this though; it's clear that the noun is improper and should be lowercase.